packages/form-js-viewer/src/render/hooks/useEffectDebugger.js

```diff
@@ -0,0 +1,59 @@
+const { useEffect, useRef, useCallback } = require('preact/hooks');
```

Comment:
Do we want to keep that?

Reply:
I think so, I use it extensively when developing more complex interactions.

Reply:
Or we move it under the `test` utils

Reply:
Oh yeah I guess that makes the most sense, so it doesn't get released.